servlet/src/test/java/io/undertow/servlet/test/dispatcher/NextServlet.java

```diff
@@ -0,0 +1,23 @@
+package io.undertow.servlet.test.dispatcher;
```

Comment:
Missing license header.

Reply:
@fl4via I've updated the changes.
Thanks.